src/js/components/search/visualizations/geo/MapLegend.jsx

```diff
@@ -34,39 +44,44 @@ export default class MapLegend extends React.Component {
     }
 
     componentDidMount() {
-        this.prepareItems(this.props);
```

Comment:
This is an interesting pattern. Might be worth preserving (I dont preserve it at all jsut a good point of discussion).

CC: @ebdabbs